test/test_torch.py

```diff
@@ -2593,6 +2593,16 @@ def _test_gather(self, cast, test_bounds=True):
         t = torch.tensor([[False, True], [True, True]])
         self.assertEqual(torch.gather(t, 1, torch.tensor([[0, 0], [1, 0]])), torch.tensor([[False, False], [True, True]]))
 
+        # Complex64, Complex128 test case
```

Comment:
I think we should just do the same thing as for float above. prob loop it for complex and float types

Reply:
https://github.com/pytorch/pytorch/blob/3b832ee2bfea70ecee65261419178165d02e3131/test/test_torch.py#L2540-L2557

Reply:
@anjali411 

Thanks for reviewing.
I fixed the complex64 and complex128 tests to do the same thing of float.